core/arch/arm/plat-stm32mp1/conf.mk

```diff
@@ -310,6 +310,10 @@ $(error CFG_STM32MP15_HUK_BSEC_KEY and CFG_STM32MP15_HUK_BSEC_DERIVE_UID are exc
 endif
 endif # CFG_STM32MP15_HUK
 
+ifeq ($(CFG_TEE_CORE_DEBUG),y)
+CFG_STM32_DEBUG_ACCESS ?= y
+endif
```

Comment:
`CFG_TEE_CORE_DEBUG` may not be set at this point and is default enabled in _mk/config.mk`. So this change may not enable CFG_STM32_DEBUG_ACCESS while CFG_TEE_CORE_DEBUG is.

Suggestion:
```patch
-ifeq ($(CFG_TEE_CORE_DEBUG),y)
-CFG_STM32_DEBUG_ACCESS ?= y
-endif
+CFG_TEE_CORE_DEBUG ?= y
+CFG_STM32_DEBUG_ACCESS ?= $(CFG_TEE_CORE_DEBUG)
```

Reply:
```
include $(platform-dir)/conf.mk
include mk/config.mk
```

Thought the other way around... ok with your suggestion